source/developer/contribution-guide.md

```diff
@@ -35,7 +35,10 @@ Before submitting a pull request (PR), check that:
 5. Strings in user interface are included in [.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json) localization files. Files for other languages will automatically be updated through the [Mattermost Translation Server](http://translate.mattermost.com) and do not need to be included in the pull request.
 6. Change meets UX Guidelines of [Fast, Obvious, Forgiving](http://www.mattermost.org/design-principles/).
 7. If change requires user to understand a new concept or make a decision, PR for help documentation is submitted to [mattermost/docs](https://github.com/mattermost/docs).
-8. Change is thoroughly tested. If your change involves text processing, make sure to run tests in [`/tests`](https://github.com/mattermost/platform/tree/master/tests).
+8. Change is thoroughly tested. If your change involves text processing, make sure to at least run markdown loadtests in [`/tests`](https://github.com/mattermost/platform/tree/master/tests) before submitting the PR. To run the loadtests:
+    - Go to **System Console** > **Developer** and set **Enable Testing Commands** to true
+    - Run `/loadtest url test-markdown-basics.md` and follow the instructions
+    - Run `/loadtest url test-markdown-lists.md` and follow the instructions
```

Comment:
we should, forgot that was separately as well. Thanks for the catch!

PS: not sure if naming it as `test-markdown-tables` would be more clear that it's a test related to markdown